src/Anon.h

```diff
@@ -11,11 +11,11 @@
 #pragma once
 
 #include <vector>
-#include <set>
 #include <map>
 
 #include "Reporter.h"
-#include "net_util.h"
+
+using std::map;
```

Comment:
We generally try to avoid `using` statements in headers, if possible, especially for `std` types. We had a PR recently that went through and removed a bunch of uses of `using namespace std` from all of the headers.

Reply:
Agree, I'd never do that - but where I added `using` statements, it was necessary to make the code build, because those sources depended on `using` statements in other headers which are no longer included.
Removing those `using` statements and adding namespace prefixes would be the next step.
But for this PR, I wanted to keep the code untouched as much as possible, and only change the includes.

Reply:
Ok, that makes sense if you're going to follow-up and remove them later. 👍